speechbrain/lobes/models/huggingface_whisper.py

```diff
@@ -104,7 +104,8 @@ def __init__(
         self._hop_length = feature_extractor.hop_length
         self._n_samples = feature_extractor.n_samples
         self.register_buffer(
-            "_mel_filters", torch.as_tensor(feature_extractor.mel_filters)
+            "_mel_filters",
+            torch.as_tensor(feature_extractor.mel_filters, dtype=torch.float32),
```

Comment:
why are you forcing the dtype? Are there some numerical instabilities?

Reply:
It turns out that `magnitudes` has `float32` dtype and `filters` has `float64` dtype. Hence, I force the dtype to avoid the error.